op-devstack/dsl/l2_el.go

```diff
@@ -85,3 +88,36 @@ func (el *L2ELNode) BlockRefByNumber(num uint64) eth.BlockRef {
 	el.require.NoError(err, "block not found using block number %d", num)
 	return block
 }
+
+// ReorgTriggered returns a lambda that checks that a L2 reorg occurred on the expected block
+// Composable with other lambdas to wait in parallel
+func (el *L2ELNode) ReorgTriggered(target eth.L2BlockRef, attempts int) CheckFunc {
```

Comment:
This could live in the CL node too, but since it is only using the `ethclient` API, I added it here.